gallery/src/Application.tsx

```diff
@@ -4,92 +4,90 @@
-
-function Page({ match, location }: RouteChildrenProps<{ pageName: string }>) {
-  const pageName = match ? match.params.pageName : null,
-      pageHeader = pageName || 'Home',
-      Content = pageName ? pageMappings[pageName] : Home;
-
-  useEffect(function() {
-    handleQueryParams(queryString.parse(location.search));
-  }, [location.search]);
-
-  if (Content) {
-    // Put a key on <main> so that it re-renders entirely on route change, resetting scroll position
-    return (
-      <main key={pageName || 'home'} className="nx-page-main">
-        <div className="nx-page-title">
-          <h1 className="nx-h1">
-            {pageHeader}
-          </h1>
-        </div>
-        <Content/>
-      </main>
-    );
-  }
-  else {
-    // unknown page
-    return <Redirect to="/" />;
-  }
-}
+import React from 'react';
+import {
+  faAtom,
+  faBiohazard,
+  faBolt,
+  faCrow,
+  faPlaceOfWorship,
+  faQuestionCircle
+} from '@fortawesome/free-solid-svg-icons';
+import {
+  NxStatefulNavigationSidebar,
+  NxNavigationSidebarContent,
+  NxNavigationSidebarLinks,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
 
 function Application() {
   return (
-    <Router>
-      <PageHeader />
-      <div className="nx-page-content">
-        <Switch>
-          <Route exact path="/PageLevelAlertExample">
-            <NxLoadWrapperPageLevelExample/>
-          </Route>
-          <Route>
-            <aside className="nx-page-sidebar" id="gallery-sidebar">
-              <GalleryNav />
-            </aside>
-            <Switch>
-              <Route path="/pages/:pageName" component={Page} />
-              <Route exact path="/" component={Page} />
-
-              {/* Special cases, these examples need their own page separate from their documentation */}
-              <Route exact path="/NxViewportSizedExample">
-                <SectionScrollingWrapper>
-                  <NxViewportSizedExample />
-                </SectionScrollingWrapper>
-              </Route>
-              <Route exact path="/NxViewportSizedExpandingExample">
-                <SectionScrollingWrapper>
-                  <NxViewportSizedExpandingExample />
-                </SectionScrollingWrapper>
-              </Route>
-              <Redirect to="/" />
-            </Switch>
-          </Route>
-        </Switch>
+    <div id="temporary-navigation-sidebar-example" className="nx-page-content nx-page-content--full-width">
+      <NxStatefulNavigationSidebar isDefaultOpen={true}
+                                   toggleOpenIcon={faBiohazard}
+                                   toggleCloseIcon={faCrow}
+                                   logoText="nexus lifecycle"
+                                   logoLink="/home">
+        <NxNavigationSidebarLinks>
+          <a href="/alpha" className="selected">
+            <NxFontAwesomeIcon icon={faAtom} fixedWidth />
+            <span>Link Name Alpha</span>
+          </a>
+          <a href="/beta">
```

Comment:
I feel like these all need classes

Reply:
> I feel like these all need classes

Classes for what?

Reply:
> I feel like these all need classes. I know you've got them all styled using descendant `a` selectors but our standard way of doing things is to give everything classes and minimize selector nesting in the SCSS

My intention when styling them that way was to minimize the requirements we put on consumer code.

Reply:
> My intention when styling them that way was to minimize the requirements we put on consumer code.

Our consistent way of doing things is to put classes on everything that needs to be styled.  No reason to deviate here.

Reply:
> I feel like these all need classes. I know you've got them all styled using descendant `a` selectors but our standard way of doing things is to give everything classes and minimize selector nesting in the SCSS

Done in b7f12c838b3c85d2eb0e03023c9ad22ea16f258c